canonical/store-components
# Fallback package title

## Done
Provided a fallback for when the display name doesn't exist

## QA
- Go to https://store-components-76.demos.haus/?path=/story/charmcard--lists
- Check that the first card has a title
